Multisig/UI/App/ViewControllerFactory.swift

```diff
@@ -17,7 +17,14 @@ enum ViewControllerFactory {
     static func rootViewController() -> UIViewController {
         let tabBarVC = MainTabBarViewController()
 
-        if !AppSettings.termsAccepted {
+        if let updateViewController = App.shared.updateController.appUpdateStatus() {
```

Comment:
it is strange to have the method named `appUpdateStatus` to return a view controller 🤪